spec/hutch_spec.rb

```diff
@@ -83,5 +83,11 @@
       Hutch.publish(*args)
     end
   end
+
+  # only works on `rspec spec/hutch_spec.rb` will be green.
+  # i don`t know why the Hutch::CLI is loaded when `rspec spec`
```

Comment:
I see this file, if this file is effect this that means when i run `rspec spec` it`s load all *_spec.rb into memery at once so the 'hutch/cli' is loaded before the `spec/hutch_spec.rb` run.